Add blog post that introduces readmap view (#320)
kyma-project/website: content/blog-posts/2019-05-20-transparency-through-roadmap/index.md

---
title: "Transparency and openness through a clear vision and roadmap"
author:
  name: "Lukasz Gornicki, Community Manager and Core&Supporting Capability Owner"
tags:
  - community
---

> Recently we released a new view in our website called [Roadmap](https://kyma-project.io/roadmap/) that's main purpose is to raise transparency of project direction and to enable contribution of the features.

Reviewer: ```suggestion
Recently, on our website, we released a new view called [Roadmap](https://kyma-project.io/roadmap/). Its main purpose is to raise the transparency of the project direction and enable features contribution.
```

---
title: "Transparency and openness through clear vision and roadmap"
author:
  name: "Lukasz Gornicki, Community Manager and Core&Supporting Capability Owner"
tags:
  - community
---

> Recently we released a new view in our website called <a href="/roadmap" target="_blank">Roadmap</a> that's main purpose is to raise transparency of project direction and to enable contribution of the features.

Reviewer 2: ``` markdown
Recently we released a new view in our website called [Roadmap](https://kyma-project.io/roadmap/) that's main purpose is to raise transparency of project direction and to enable contribution of the features.
```

<!-- overview -->

## Choosing the right tool

Choosing the right open source tool is not easy. There are many different factors you need to take into account before you make the right decision. Luckily internet if full of ideas and suggestions on how to do that, like article [How to Choose the Best Open Source Software](https://towardsdatascience.com/how-to-choose-the-best-open-source-software-b1cbbe4f6398).

> > Remember that the amount of commits, issues, and pull-requests (PRs) can be a signal of investment and commitment to a project.

Author: this is a quote from the article ;) should stay as it is, this is why I used markdown quote

> > Remember that the amount of commits, issues, and pull-requests (PRs) can be a signal of investment and commitment to a project.

Reviewer: Got it now!

We wanted to ease the things for the Kyma community. We identified in Kyma different capabilities it has and shared our goal and visions for each. Thanks to such approach each capability has its own roadmap but created in sync with other capabilities, so it creates one overview of all the things that are going to happen in Kyma in the short and long term. We combined all in one view 

> called <a href="/roadmap" target="_blank">Roadmap</a>.

Reviewer 2: ``` markdown
called [Roadmap](https://kyma-project.io/roadmap/).
```

> Thanks to this view you can clearly identify areas of Kyma that are the most important for you and see if planned development goes the direction you expect. If something is not the way you expected you can always share with us your opinion on our [Core SIG meetings](https://github.com/kyma-project/community/tree/master/sig-and-wg/sig-core), especially if you notice we are not fully transparent for a specific capability and its roadmap is not properly maintained. The other option to make things better is to join us and contribute.

Reviewer: ```suggestion
Thanks to this view you can clearly identify the areas of Kyma that are the most important for you, and see if the planned development goes in the direction you expect. If something is not the way you wanted, you can always share your opinion with us at our [Core SIG meetings](https://github.com/kyma-project/community/tree/master/sig-and-wg/sig-core). Contact us especially if you notice that we are not fully transparent regarding a specific capability, and its roadmap is not properly maintained. Another way to make things better is to join us, and contribute.
```

## Easier contribution

> Being an open source user for a few years already, I noticed on many different channels situation where people want to start a contribution to a project, but they do not know what they could contribute. It is quite common that people ask `I want to contribute to this project, what feature would be the most interesting for people`. This is another reason why we worked on this new view on our website. We don't want to search and invent ideas for contribution, all that you need is there, in our roadmap. You can just pick the feature that is set for future development and go forward with proposal and implementation. This way you know that you contribute something that is important for Kyma and its future.

Reviewer: ```suggestion
Being an open source user for a few years now, I noticed on many different channels that people want to start a contribution to a project, but they do not know what they could contribute. There is a common question: `I want to contribute to this project, what feature would be the most interesting for others?` This is yet another reason why we worked on this new view on our website. We don't want you to search and invent ideas for contribution. All you need is already there, in our Roadmap. You can just pick a feature that is planned for development, and go forward with a proposal and implementation. This way you know that you contribute to something that is important for Kyma and its future.
```
